docs/topics/proc_deploying-an-application-to-standalone-rhel.adoc

```diff
@@ -0,0 +1,121 @@
+// This is a parameterized module. Parameters used:
+//
+// :parameter-jar-file-name: runtime-specific default target Maven build artifact name.
+// :parameter-version-runtime-plugin: runtime-specific name for the runtime version property variable in the POM file.
+// :parameter-artifact-name-runtime-plugin: runtime plugin name in the POM file.
+// :parameter-group-name-runtime-plugin: runtime plugin Maven artifact group name.
+// :parameter-runtime-name: Runtime specific atribute for substituting the approrpiate expanded rumtime name.
+//  context: used in anchor IDs to conflicts due to duplicate IDs.
+//
+// Rationale: This procedure is the same for 2 or more runtimes.
+[id='deploying-an-application-to-standalone-rhel_{context}']
+= Deploying an application to standalone Red Hat Enterprise Linux
+
+You can deploy a {parameter-runtime-name} application to Red Hat Enterprise Linux.
+
+.Prerequisites
+
+* A Maven-based {parameter-runtime-name} application.
+* RHEL 7 installed.
+* OpenJDK 8 installed.
+* Maven 3.5 or later installed.
+
+
+.Procedure
+
+. Ensure that you specify the link:https://maven.repository.redhat.com/[Red Hat Maven repository] in the `pom.xml` of your project.
```

Comment:
I think in Vert.x case this step is unnecessary because is pretty well described in the previous [Creating an application sub-section](https://github.com/fabric8-launcher/launcher-documentation/blob/749cdcc066149349825620c05b713e9130ca1e18/docs/topics/proc_creating-an-application.adocl), but if you want to keep it you should add this step to [Deploying an application to OpenShift sub-section](https://github.com/fabric8-launcher/launcher-documentation/blob/749cdcc066149349825620c05b713e9130ca1e18/docs/topics/proc_deploying-an-application-to-openshift.adoc) too.

Reply:
I think it is the same for other runtimes WDYT @Ladicek  @geoand  @liborfuka?

Reply:
Definitely.

Reply:
:+1:

Reply:
@akoniar @geoand @Ladicek Thank you for pointing this out. When viewed in context of the other procedures on application development  that we have in the runtime guides, this step is indeed superfluous. I have therefore removed it and replaced with a prerequisite with a link pointing to the _Developing an application_ section that contains the POM with the repository configuration example. WDYT?

Reply:
Prerequisite also makes sense, but I'd personally rather have all the important info in one place. But I don't mind either way.

Reply:
I agree to keep it like that.